### ISSUE TITLE
Low contrast in Card on "Experience and education requirements" in the job poster template

### ISSUE THREAD

**Issue by Elisa0122** [GCTC-NTGC/gc-digital-talent#6251]
## ❌ Current behaviour (issue)

Contrast in  "Experience and education requirements" in the job poster template fails WCAG 1.4.3 AA contrast criteria. The white foreground text in the title does provide contrast with the aqua background.

**WCAG 2.1 Criteria:** 1.4.3: The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for large text, text or images of text that are part of an inactive user interface component, or Logotypes.

### How to reproduce

1. Environment (prod or UA?)
2. Which browser and/or Assistive Technology?
3. Step-by-step instructions explaining how to reproduce the problem.

## ✔ Expected behaviour (fix)

Text is black so that it provides enough contrast for low-vision users and those who may have  colour blindness. 
### Acceptance Criteria

- [ ] Black text  provides enough contrast against the aqua background for low-vision users and those who may have  colour blindness.

## Screenshot
![image](https://user-images.githubusercontent.com/8978655/231769955-dea4bed0-dba8-461a-9389-d5a07636d4ed.png)

## Additional information (optional)

1. How does it impact users?
Low contrast can have a significant impact on users, particularly those with visual impairments or color vision deficiencies. When there is low contrast between text and background colors, it can make the text difficult to read and strain the eyes.

2. Related issues, suggestions how to fix
Using black colour instead of white for the text it provides enough contrast for low-vision users and those who may have colour blindness.
